comet/api/stream.py

```diff
@@ -516,11 +500,10 @@ async def playback(request: Request, b64config: str, hash: str, index: str):
                 },
             )
 
-        if (
-            settings.PROXY_DEBRID_STREAM
-            and settings.PROXY_DEBRID_STREAM_PASSWORD
-            == config["debridStreamProxyPassword"]
-        ):
+        if not should_try_to_proxy_stream:
+            return RedirectResponse(download_link, status_code=302)
```

Comment:
StremThru will generate a link to itself, that can proxy stream to debrid direct links.

So Comet just needs to redirect to that link.